### PR TITLE
🎩 update Properties design

### PR DESCRIPTION
**Pull request by roiLeo**
I was feeling a bit inspired
take it or leave it

### PR type

- [x] Feature
- [x] Refactoring

### What's new?

- [x] move Properties components outside unique folder
- [x] update Properties (attributes) component design since I can't stand having so much table everywhere

### Before submitting Pull Request, please make sure:

- [x] My contribution builds **clean without any errors or warnings**
- [x] I've merged recent default branch -- **main** and I've no conflicts
- [x] I've tried to respect high code quality standards
- [x] I've didn't break any original functionality
- [x] I've posted a screenshot of demonstrated change in this PR

### Screenshot

Tested on `/bsx/gallery/2444376227-3`

**before**
Screenshot 2022-08-19 at 21-10-26 3D Ball KodaDot - Kusama NFT Market Explorer

**after**
Screenshot 2022-08-19 at 21-05-23 3D Ball KodaDot - Kusama NFT Market Explorer
Screenshot 2022-08-19 at 21-04-39 3D Ball KodaDot - Kusama NFT Market Explorer

### CODE REVIEW

**Comment by yangwao** [comment]
Yes!

File: components/shared/gallery/Properties.vue [hunk 1]
```diff
@@ -1,13 +1,25 @@
 <template>
   <CollapseCardWrapper :label="$t('nft.properties.label')">
-    <b-table :data="displayAttributes" :columns="columns" />
+    <div class="columns is-multiline">
+      <div
+        v-for="attribute in attributes"
+        :key="attribute.trait_type"
```
Comment (Jarsen136) [inline components/shared/gallery/Properties.vue:6]:
```suggestion
        :key="attribute[fieldKey]"
```

Hey, there is a props value `fieldKey`

**Review by Jarsen136 — COMMENTED** [review]
Nice Design!

File: components/shared/gallery/Properties.vue (hunk 1 above)
Comment (roiLeo) [reply components/shared/gallery/Properties.vue:6]: Nice catch!
`<Properties :attributes="meta.attributes" field-key="trait_type" />` 

but why it has been introduced? won't this always be `trait_type`?

Comment (roiLeo) [reply components/shared/gallery/Properties.vue]: fixed in new commit

**Review by Jarsen136 — APPROVED** [review]

**Review by DeluneClaire — APPROVED** [review]
This is very nice! WFM:)

**Pull request merged**